module/VuFind/src/VuFind/Connection/ConsortialVuFind.php

```diff
@@ -0,0 +1,117 @@
+<?php
+
+/**
+ * Consortial VuFind API connection class.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2023.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ *
+ * @category VuFind
+ * @package  Connection
+ * @author   Maccabee Levine <msl321@lehigh.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org
+ */
+
+namespace VuFind\Connection;
+
+use Exception;
+use Laminas\Log\LoggerAwareInterface;
+
+/**
+ * Consortial VuFind API connection class.
+ *
+ * @category VuFind
+ * @package  Connection
+ * @author   Maccabee Levine <msl321@lehigh.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org
+ */
+class ConsortialVuFind implements
```

Comment:
Maybe it makes sense to keep this class named ExternalVuFind or RemoteVuFind or VuFindAPI. The ConsortialVuFind recommendation module name makes sense because it's presenting information specific to the consortial use case... but this is a general-purpose utility class for connecting to external VuFind instances, and presumably if you made other VuFind-driven recommendation modules for different use cases, they would all still share use of this utility.

Reply:
I think that's a good call, and I should rename the .ini file also since it's mostly about the connection and could be used with other purposes.  I like either 'external' or 'remote', not just 'VuFindAPI' since that could be confused for something about VF as the server component.  Do you have a preference?

Reply:
For now I decided to go with 'external' since that was my first instinct.